service-wfs-client/src/main/java/org/oskari/service/wfs/client/OskariWFS110Client.java

```diff
@@ -37,17 +41,17 @@ private OskariWFS110Client() {}
      * @return SimpleFeatureCollection containing the parsed Features, or null if all fails
      */
     public static SimpleFeatureCollection tryGetFeatures(String endPoint, String user, String pass,
-            String typeName, ReferencedEnvelope bbox, CoordinateReferenceSystem crs, Integer maxFeatures) {
+            String typeName, ReferencedEnvelope bbox, CoordinateReferenceSystem crs, Integer maxFeatures, Filter filter) {
```

Comment:
Should we have an overloaded version of the method without the filter that would call this one with null value.

Reply:
Added